botocore/utils.py

```diff
@@ -141,6 +140,38 @@
     "tagging": "resource-groups-tagging-api"
 }
 
+# Vendoring ipv6 validation regex patterns from urllib3
```

Comment:
Minor nit: Let's use change ipv6 to IPv6. Otherwise, I think we're looking good.